<launch>
<!-- Launch file for Livox AVIA LiDAR -->

  <arg name="rviz" default="true" />
  <rosparam command="load" file="$(find std_detector)/config/config_ntu.yaml" />
  <node 
		pkg="std_detector" 
		type="relocalization_kitti" 
		name="relocalization_kitti" 
		output="screen" 
	/>

  <!-- descriptor database generation -->
  <param name="generate_descriptor_database" type="bool" value = "false"/>
  <param name="generate_lidar_path" type="string" value = "/data/KITTI_Benchmark/Odometry/data_odometry_velodyne/dataset/sequences/00/velodyne/"/>
  <param name="generate_pose_path" type="string" value = "/data/std_example/kitti00.txt"/>

  <!-- relocalization -->
  <param name="descriptor_file_path" type="string" value = "/data/KITTI_Benchmark/Odometry/data_odometry_velodyne/dataset/sequences/00/saved_descriptors/"/>
  <param name="localization_lidar_path" type="string" value = "/data/KITTI_Benchmark/Odometry/data_odometry_velodyne/dataset/sequences/00/velodyne/"/>
  <param name="localization_pose_path" type="string" value = "/data/std_example/kitti00.txt"/>
  <!-- <param name="reference_gt_path" type="string" value = "/home/samsung/NTU_carpark/ntu_day_02/pose_inW.csv"/> -->

  <!-- global map for visualizatin -->
  <!-- <param name="global_map_path" type="string" value = "/home/samsung/NTU_carpark/Global Map/GlobalFullMap.pcd"/> -->

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find std_detector)/rviz_cfg/demo.rviz" />
  </group>

</launch>